lib/GetData.js

```diff
@@ -0,0 +1,17 @@
+const axios = require('axios');
```

Comment:
I think we need to rethink this method. UX is more important than code sharing & users shouldn't have to deal with raw html documents.
In a case where a user wishes to search songs and get their lyrics/albumArt manually, they have to call three methods in a sequence, which is not very good.
```
let results = await searchLyrics(options)
let data = await getData(results[0].path)
let lyrics = await getLyricsFromPath(data)
```
should be just
```
let results = await searchLyrics(options);
let lyrics = await getLyricsFromPath(results[0].path)
```

Similarly,

```
let results = await searchLyrics(options)
let data = await getData(results[0].path)
let albumArt = await getAlbumArtFromPath(data)
```
should be
```
let results = await searchLyrics(options)
let albumArtUrl = await getAlbumArtFromPath(results[0].path)
```

What are your thoughts?

Reply:
I agree. Now that I've thought about it for a while, it seems more logical. I will fix this in the next commit.

Reply:
Added in 8be3119d7d4dd6c04b9ad27adfd08dadc1fb185e